generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-ftp/pom.xml

```diff
@@ -59,13 +59,13 @@
     <dependency>
       <groupId>org.apache.ftpserver</groupId>
       <artifactId>ftpserver-core</artifactId>
-      <version>${quarkus-cassandra-client.version}</version>
+      <version>1.1.1</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.ftpserver</groupId>
       <artifactId>ftplet-api</artifactId>
-      <version>${quarkus-cassandra-client.version}</version>
+      <version>1.1.1</version>
```

Comment:
There is no reason for this commit really. As I mentioned in my previous comment, it's harmless although still definitely wrong. It will re-appear on every platform build.